packages/react-core/src/components/DragDrop/examples/DragDrop.md

```diff
@@ -1,175 +1,39 @@
 ---
 id: Drag and drop
 section: components
-propComponents: [
-  DragDrop,
-  Draggable,
-  Droppable,
-  DraggableItemPosition
-]
+propComponents: [DragDrop, Draggable, Droppable, DraggableItemPosition]
 beta: true
 ---
 
 You can use `DragDrop` to move items in or between lists. `DragDrop`s should contain `Droppable`s which contain `Draggable`s.
```

Comment:
I know this is outside of the scope of this PR, but I think we should change this wording a little because of the way renders in the docs.  I would say 

You can use `DragDrop` component to move items in or between lists. The `DragDrop` component should contain `Droppable` components  which contain `Draggable` components.